include/struct_defs/pokemon_personal_data.h

```diff
@@ -0,0 +1,44 @@
+#ifndef POKEPLATINUM_STRUCT_02075874_H
+#define POKEPLATINUM_STRUCT_02075874_H
+
+/**
+ * @brief Pokemon Personal data structure. This contains data that is the same across all pokemon of the same species/form
+ */
+typedef struct PokemonPersonalData {
+    u8 baseHp; //!< The pokemons base HP stat
+    u8 baseAtk; //!< The pokemons base Attack stat
+    u8 baseDef; //!< The pokemons base Defense stat
+    u8 baseSpeed; //!< The pokemons base Speed stat
+    u8 baseSpAtk; //!< The pokemons base Special Attack stat
+    u8 baseSpDef; //!< The pokemons base Special Defense stat
+    u8 type1; //!< The pokemons first type
+    u8 type2; //!< The pokemons second type
+    u8 catchRate; //!< The pokemons catch rate
+    u8 baseExp; //!< A factor in determining experience yield when defeating this pokemon
+    u16 evHpYield : 2; //!< How many HP EVs will be gained when defeating this pokemon
+    u16 evAtkYield : 2; //!< How many Attack EVs will be gained when defeating this pokemon
+    u16 evDefYield : 2; //!< How many Defense EVs will be gained when defeating this pokemon
+    u16 evSpeedYield : 2; //!< How many Speed EVs will be gained when defeating this pokemon
+    u16 evSpAtkYield : 2; //!< How many Special Attack EVs will be gained when defeating this pokemon
+    u16 evSpDefYield : 2; //!< How many Special Defense EVs will be gained when defeating this pokemon
+    u16 padding : 4;
+    u16 item1; //!< Common held item when this pokemon is encountered in the wild
```

Comment:
Add a linebreak here for some visual clarity.

```suggestion
    u16 padding : 4;
    
    u16 item1; //!< Common held item when this pokemon is encountered in the wild
```